docs/README-submodule-api.md

```diff
@@ -124,6 +124,7 @@ Parameters:
   after `@` sign, in which case we check if the sums match. For other orchestrator sub-modules
   it maybe a path, or some other resource handler.
 * `path` -- path to a file holding the data
+* `current_url` -- in case of [deep deltas](#deep-binary-delta-of-images) we need the current url to apply them
```

Comment:
it is passed in the last argument to the LOAD operation, last line of the test. it is just hardcoded there, since we know which image we are playing with.